README.md

```diff
@@ -76,6 +76,26 @@ maturin develop
 
 Then run `python3` with the above Python code!
 
+### WASM
```

Comment:
I think this should be in a README inside the wonnx-wasm repository to avoid this README to be packed with to many things.

We can then add those documentation to docs.rs using:
```
#[doc = include_str!("../README.md")]
```

Only putting a light reference here.

Reply:
Agreed, the main README should say something like "In order to use from JS, just `npm install wonnx-wasm`" (i.e. aimed at end users) whereas `wonnx-wasm/README.md` should describe the build process for developers.

Reply:
Did not know about that include_str trick by the way!